build_pip_pkg.sh

```diff
@@ -16,44 +16,26 @@
 set -e
 set -x
 
-PLATFORM="$(uname -s | tr 'A-Z' 'a-z')"
-
 PIP_FILE_PREFIX="bazel-bin/build_pip_pkg.runfiles/__main__/"
 
-if [[ ${PLATFORM} == "darwin" ]]; then
-    READLINK="greadlink"  # from pkg `coreutils` in brew/macports in Mac OS X
-else
-    READLINK="readlink"
-fi
-
+function abspath() {
+  cd $(dirname $1)
```

Comment:
could you change back the directory before returning result?

Reply:
Done. thx.